securesign/cosign
# feat: customise the TAS successful auth landing page

This pr adds the changes necessary to update the landing page when a user authenticates with keycloak.

## Testing 

1. Apply the overlay files 
2. **Make sure you are using a fresh build of cosign created after the overlay files are applied**
3. Stand up a cluster 
4. Using key cloak and the new cosign binary sign and verify an image  
5. You should be brought to the new landing page instead of the old one 

## Before
![image](https://github.com/securesign/cosign/assets/59924001/90e70e56-bd5b-4789-a865-0395aca3c161)

## After
![Screenshot from 2023-11-02 15-41-38](https://github.com/securesign/cosign/assets/59924001/ac8b8793-b7f0-44bd-9c8d-5ab9bbcee192)
